1-exercises/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,28 @@
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach((num)=> {
+    if (num % 3 === 0) {
+      console.log("Fizz");
+    } else if (num % 5 === 0) {
+       console.log("Buzz");
+    } else if (num % 3 === 0 && num % 5 === 0) {
+       console.log(FizzBuzz);
+    } else {
+       console.log(num);
+    }
+  })
```

Comment:
good use of the remainder operator here, but have a think about the ordering of your if/else statements. When 15 is passed, I would expect this to log Fizzbuzz, however at the moment I would be getting Fizz

Reply:
Hi, I have corrected the code. So I have provided comparison operator of (num % 3 === 0 && num % 5 === 0)  first . Now its working. thanks.